simms/skymodel/skymods.py

```diff
@@ -117,18 +122,18 @@ def computevis(srcs, uvw, freqs, ncorr, mod_data=None, noise=None, subtract=Fals
         n_term = np.sqrt(1 - el*el - em*em) - 1
         arg = uvw_scaled[0] * el + uvw_scaled[1] * em + uvw_scaled[2] * n_term
         if source.emaj in [None, "null"] and source.emin in [None, "null"]:
-            vis += source.spectrum * np.exp(-2 * np.pi * 1j * arg)
+            vis += source.spectrum[0] * np.exp(-2 * np.pi * 1j * arg)
```

Comment:
This line, along with lines 136 and 141 cannot be right. Since this is full-stokes, you need to use the brightness matrix's relation to the feed corrections, i.e,

$xx = I+Q$
$xy = U+iV$
$yx = U-iV$
$yy = I-Q$